Description: remove compiler-builtins feature.
 The feature fails to build and the ftpmasters are rejecting new rust
 feature packages.
Author: Peter Michael Green <plugwash@debian.org>

Index: miniz-oxide/Cargo.toml
===================================================================
--- miniz-oxide.orig/Cargo.toml
+++ miniz-oxide/Cargo.toml
@@ -42,10 +42,6 @@ name = "miniz_oxide"
 version = "1.0"
 default-features = false
 
-[dependencies.compiler_builtins]
-version = "0.1.2"
-optional = true
-
 [dependencies.simd-adler32]
 version = "0.3"
 optional = true
